package.json

```diff
@@ -19,6 +19,7 @@
     "test:unit:coverage": "karma start --single-run --reporters spec,coverage",
     "test:functional": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe chrome",
     "test:functional:watch": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" ./scripts/watchFunctionalTests.js --browsers chrome",
+    "test:functional:debug": "EDGE_BASE_PATH=\"ee-pre-prd\" ALLOY_ENV=\"int\" testcafe --inspect-brk chrome -T \"Test C28758.*\"",
```

Comment:
I guess this is a non-intentional change. You could also change inside functional test `test.only` and run `test:functional:watch`, this will run only that specific test in watch mode.

Reply:
@ninaceban No, this is actually an intentional change, as with the Testcafe Live Runner used in the `watch` script, there's no way to start a proper Inspector debug server, so that the test can be debugged directly in the IDE.

Reply:
@ninaceban I have also moved the test focusing to this external `debug` script, instead of using `test.only`, as ESLint warns you not to do it: `ESLint: don't focus tests(ban/ban)` as it's easier to forget to remove it once you set it in the test, which is not an issue when you have a separate debug script.

Reply:
As far as I can understand this is a debug configuration only for one test case. You could have this configured locally without this being committed to the repo.

Reply:
In addition to the reasons I mentioned above, the idea is to commit it to the repo so that there's an explicit script for debugging a single functional test, and any new contributors don't have to rely on tacit knowledge on how to debug a single test, losing time for looking up how to focus a single test in our particular testing framework, but can instead do it quickly just taking a look at the script and adjusting it to their needs.

Reply:
@jonsnyder Fixed as suggested and added instructions to the Wiki